Added Moonrise-class Emergency Response Cruiser (Non-voteable)

LittleBuilderJane opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Adds the Moonrise-class Cruiser.
**Screenshots**
<!-- If applicable, add screenshots to showcase your PR. If your PR is a visual change, add
screenshots or it's liable to be closed by maintainers. -->
![Moonrise Screenshot](https://github.com/LittleBuilderJane/screenshotsSS14/blob/main/Moonrise.png?raw=true)
**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
-->

:cl: Schiene
- add: Added a new non-votable map, Moonrise-class Emergency Response Cruiser.

LittleBuilderJane commented:
The Picture is slightly outdated, the shuttle DOES have windows.

metalgearsloth reviewed (changes requested):
A bunch of pieces are going to detach immediately whenever grid splitting is coded.

LittleBuilderJane commented:
> A bunch of pieces are going to detach immediately whenever grid splitting is coded.



> A bunch of pieces are going to detach immediately whenever grid splitting is coded.

Updated and fixed.

metalgearsloth commented on Resources/Prototypes/Maps/game.yml:
```diff
@@ -41,6 +41,10 @@
     prefixCreator: 'VG'
   mapPath: Maps/packedstation.yml
   minPlayers: 15
+  conditions:
+    - !type:HolidayMapCondition
+      inverted: true
+      holidays: [ "Christmas" ]
   overflowJobs:
```
?

LittleBuilderJane replied:
Dunno where that came from, but it's most definitely NOT my code. I didn't write that.
Removed it.

metalgearsloth commented on Resources/Prototypes/Maps/game.yml:
```diff
@@ -86,3 +90,49 @@
     ResearchDirector: [ 1, 1 ]
     Botanist: [ 1, 1 ]
+
+- type: gameMap
+  id: sunrise
+  mapName: 'Sunrise'
+  mapNameTemplate: '{0} Sunrise {1}'
+  nameGenerator:
+    !type:NanotrasenNameGenerator
+    prefixCreator: 'SS'
+  mapPath: Maps/sunrise.yml
+  minPlayers: 0
+  maxPlayers: 8
+  overflowJobs: []
+  availableJobs:
+    Bartender: [ 1, 1 ]
+    Captain: [ 1, 1 ]
+    ChiefEngineer: [ 1, 1 ]
+    StationEngineer: [ 1, 1 ]
+    ChiefMedicalOfficer: [ 1, 1 ]
+    MedicalDoctor: [ 1, 1 ]
+    ResearchDirector: [ 1, 1 ]
+    Botanist: [ 1, 1 ]
+
```
Not a map

LittleBuilderJane replied:
That's from another branch of mine, dunno how it got there.
Removed.

metalgearsloth commented:
Requires https://github.com/space-wizards/space-station-14/pull/6392
